### DISCUSSION TITLE
Gene set enrichment analysis with ArchR ?

### DISCUSSION THREAD

**Discussion by Nidane** [GreenleafLab/ArchR#1727] [Feature Requests]
Hi team,
 
Thanks a lot for such an amazing tool! 

Just wonder - as did not come across other issues or discussions yet - is there a possible way to use ArchR to carry out gene set enrichment analysis - guess could use the Gene Score etc. for this ? 

Thanks

**Comment by rcorces** [comment]
There is no built-in functionality for this. you would get your diff genes in whatever way makes sense for your analysis and take that gene list to an external tool to perform GSEA.
